src/components/Home.js

```diff
@@ -24,11 +25,12 @@ class Home extends Component {
 
   componentWillMount() {
     const { cookies } = this.props;
+
     if (!this.state.authChecked) {
       //this.checkAuth()
       // check if the cookie is stored
       this.setState({
-        isLoggedIn: cookies.get("RubricsApp"),
+        isLoggedIn: cookies.get("RubricsApp") !== "",
```

Comment:
I think we also need to also if it is equal to "undefined".
It's good that we also check it's empty string right now

Reply:
We can make the changes in the next code changes

Reply:
We still need to check for undefined